libs/sr-db/src/Database/DataManager.ts

```diff
@@ -0,0 +1,33 @@
+import type { Database } from '@genshin-optimizer/database'
+import { DataManagerBase } from '@genshin-optimizer/database'
+import type { ISrObjectDescription } from '@genshin-optimizer/sr-srod'
+import type { ISroDatabase } from '../'
+import type { ImportResult } from './exim'
+export class DataManager<
+  CacheKey extends string,
+  DataKey extends string,
+  CacheValue extends StorageValue,
+  StorageValue,
+  DatabaseType extends Database
+> extends DataManagerBase<
+  CacheKey,
+  DataKey,
+  CacheValue,
+  StorageValue,
+  DatabaseType
+> {
+  exportSROD(sro: Partial<ISrObjectDescription & ISroDatabase>) {
+    const key = this.dataKey.replace('sro_', '')
```

Comment:
@frzyc please verify this is safe

Reply:
should be fine? this should only change the export format. you should check whether the exported format adheres to SROD